packages/camera/camera_android_camerax/example/pubspec.yaml

```diff
@@ -14,7 +14,8 @@ dependencies:
     # The example app is bundled with the plugin so we use a path dependency on
     # the parent directory to use the current plugin's version.
     path: ../
-  camera_platform_interface: ^2.2.0
+  camera_platform_interface: ^2.6.0
+
```

Comment:
Please remove this line (see other related comments).

Reply:
removed